packages/asset/contracts/AssetCreate.sol

```diff
@@ -99,11 +99,19 @@ contract AssetCreate is
             "Invalid signature"
         );
 
+        // function generateTokenId(
+        // address creator,
+        // uint8 tier,
+        // uint16 creatorNonce,
+        // uint16 revealNonce,
+        // bool bridged
+        // ) internal pure returns (uint256 tokenId)
+
         uint256 tokenId = TokenIdUtils.generateTokenId(
             creator,
             tier,
             ++creatorNonces[creator],
-            revealed ? 1 : 0,
+            0, // revealNonce is always 0 for a newly created asset
```

Comment:
To do: is this how we want revealNonce to work, i.e. 0 for not revealed, 1 for revealed?

Reply:
To do: Make sure similar changes (if any) are also made to MockMinter.sol, which is used for generating tokenIds for reveal tests